CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,111 @@
-# Contributor Covenant Code of Conduct
+# Code of Conduct at nf-core (v1.0)
```

Comment:
Do you want to keep nf-core here and in the rest of the text?

Reply:
yeah I guess for this file we would just keep the same Code of Conduct as nf-core